jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml

```diff
@@ -162,6 +162,7 @@
                 export GKE_CREATE_FLAGS="--max-nodes-per-pool=1000"
                 export CLOUDSDK_CONTAINER_USE_CLIENT_CERTIFICATE=True
                 export CLOUDSDK_API_ENDPOINT_OVERRIDES_CONTAINER="https://staging-container.sandbox.googleapis.com/"
+                export KUBE_GKE_IMAGE_TYPE="gci"
```

Comment:
How is this related to:
https://github.com/kubernetes/test-infra/pull/664/files#diff-3945c589c3c116b9fd9824f0783f1905R184